### ISSUE TITLE
The "packetevents" library is not up to date

### ISSUE THREAD

**Issue by braceyourself07** [Summiner/ServerPatches#14]
I'm not a dev so I'm not sure if I'm explaining it properly, but I believe the "packetevents" library is not up to date.  I recently added ViaVersion/ViaBackwards to allow support for 1.20.5 and it is spamming a packetevent error that the Via devs say is due to a library in a plugin that is not up to date.  I have isolated it to Server-Patches.jar and even with the March 2024 build, it still spams the packetevents error in the console.  Is there any fix to this?  Let me know if there is any more specific information that I could provide.

Basically packetevents is not fetching the client versions correctly.

**Comment by Summiner** [comment]
Could you please provide some logs of the error it's throwing? the latest commit uses the latest version of PacketEvents which (seems) to be fine so I think it's an issue with my code. Thank you ^^

**Comment by braceyourself07** [comment]
Just a repeating message in console, no stack traces.
![errors](https://github.com/Summiner/ServerPatches/assets/4984394/b85a5613-90ec-41c4-ae2f-9c4d73d96a72)

The error is no longer spammed when Server-Patches.jar is disabled.

This was recommended by one of the helpers on the ViaVersion support website:

```
If your logs are spammed with Unable to grab ViaVersion client version for player!, that's because the library "packetevents" in one of your plugins is doing something dumb. In order to find which of your plugins contains this library, run the following bash script and make sure the plugin(s) in question are up-to-date:
#!/bin/bash
for jar_file in *.jar; do
    if unzip -l "$jar_file" | grep -q '^.*packetevents/$'; then
        echo "Update/complain to $jar_file"
    fi
done
```

The script is what I used to identify Server-Patches.jar as the only plugin I was using that contained a reference to packetevents.  

Hopefully this context helps!

**Comment by Summiner** [comment]
There is currently a commit of PacketEvents that fixes this issue, later I'm submit a temporary commit that uses that version and the next release if ServerPatches will be pushed when they push a new release.

**Comment by Kas-tle** [comment]
Just FYI while debugging this issue I also noticed that the path to the main class in plugin.yml seems to be incorrect... likely was never changed after the gradle refactor.

**Comment by Summiner** [comment]
The latest commit should fix this issue, I'll be cleaning up the issue mentioned above by @Kas-tle along with a few other things and should have a new release out some time tomorrow. Thank you both for your help ^^
